install/PackageConfig.cmake.in

```diff
@@ -53,4 +55,4 @@ endif()
 
 if(NOT _ZENOHCXX_BACKEND_FOUND)
 	message(FATAL_ERROR "Failed to detect zenoh-cpp backend, you need to have either zenoh-c or zenoh-pico installed" )

```

Comment:
Won't the second call here cause the FATAL_ERROR ?

Reply:
`_ZENOHCXX_BACKEND_FOUND` is not used in other places, we can remove it

Reply:
Ok, I got rid of the `_ZENOHCXX_BACKEND_FOUND` variable.